app.rb

```diff
@@ -98,6 +98,28 @@ def require_login(redirect_after_login=nil)
     else
       redirect "/login/?next_page=#{redirect_after_login}"
     end
+  elsif SS_ID == 8
```

Comment:
We probably want some check here for super users / admins. If I am a super user, I shouldn't have to accept the user agreement to edit a tool record.

Reply:
Updated [6c87ef1](https://github.com/unl/reservations/pull/99/commits/6c87ef1f5abdb4c7f13621c6d7b5aa348e1f2f48)